string actual_place
pose actual_pose
---
 SemanticalPoint final_message
 
